### DISCUSSION TITLE
Are these too many daily requests?

### DISCUSSION THREAD

**Discussion by YO-SC** [supabase/supabase#23007] [Questions] [answered]
Hello, I am developing web applications in NextJS. I follow the official NextJS guides on how to use Supabase with NextJS, specifically the [user management example](https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs). 

Interestingly while I continue development in web apps I notice a good portion of requests in my Supabase dashboard. 

![image](https://github.com/supabase/supabase/assets/48807115/9315e327-583f-49f0-8377-88635b265cb7)

I am here to ask if this is normal in the Supabase realm. 

## My implementation on NextJS v14+

For those wondering, the way I do things in NextJS (after following the [user management example](https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs)) is to check if the current user has access to certain pages. For further context, I am using the [App Router](https://nextjs.org/docs/app/building-your-application/routing). So i use the following code on the NextJS pages that require it, before returning the page component:

```ts
  const supabase = createClient()
  const {
    data: { user },
  } = await supabase.auth.getUser()
 ```
 
 The `createClient()` in particular comes from `@/utils/supabase/server`. The code comes from [the example](https://supabase.com/docs/guides/getting-started/tutorials/with-nextjs?queryGroups=language&language=ts#supabase-utilities).

## My question

So again, is having those amounts of requests normal given the implementation? Are there any ways in which I can improve my implementation? Any help is welcomed, thanks y'all.

**Comment by GaryAustin1** [comment, marked as answer]
While you are using the dashboard there are request to auth and the database for heath checks.

If you go to the report page and API requests you can see a breakdown.
![image](https://github.com/supabase/supabase/assets/54564956/f3295c8e-572d-4ba3-be61-9b34385b30f0)

**Reply by GaryAustin1** [reply]
You should respond to the original answer not create a new answer.

Yes to your last question/understanding.  What I meant to say there is just being in the dashboard generates those two requests.

**Reply by YO-SC** [reply]
Ah kk, sorry about that 😅 

Also alright 👍🏼 , and thank you so much for your time and responses.

**Comment by YO-SC** [comment]
Interesting, the numbers over there seem normal:

![image](https://github.com/supabase/supabase/assets/48807115/4b5cb6ac-0f48-4065-944d-ccbfe017223d)

On the dashboard looks like this:

![image](https://github.com/supabase/supabase/assets/48807115/db687928-84cc-4bbb-8e61-9356e1768308)

Time has passed so the numbers are different since I opened the discussion. Anyhow, I wonder the difference between all this numbers. I am relatively new to the backend world, I am more of a frontend guy 😄 .

**Reply by YO-SC** [reply]
I am aware/think that by using the Supabase dashboard and tools itself count to the requests. If I am wrong feel free to correct me 😇.
